src/main/app/src/assets/i18n/en.json

```diff
@@ -290,14 +290,22 @@
   "indicaties": "indicators",
   "indicaties.-kies-": "- Select an indication -",
   "indicaties.-geen-": "- No indication -",
-  "indicatie.GEBRUIKSRECHTEN": "Has usage rights",
+  "indicatie.GEBRUIKSRECHT": "Has usage rights",
```

Comment:
Ideetje: Zet de afkorting er in hoofletters in. Dus

Has Usage rights
Is Reopend
Has been eXtended

Reply:
ik vond de x wel leuk, maar heb het wat consistenter gemaakt, door de letter E te gebruiken